pkg/asset/machines/openstack/machines.go

```diff
@@ -78,36 +78,57 @@ func Machines(clusterID string, config *types.InstallConfig, pool *types.Machine
 	return machines, nil
 }
 
-func provider(clusterID string, platform *openstack.Platform, mpool *openstack.MachinePool, osImage string, az string, role, userDataSecret string, trunk string) (*openstackprovider.OpenstackProviderSpec, error) {
+func provider(clusterID string, platform *openstack.Platform, mpool *openstack.MachinePool, osImage string, az string, role, userDataSecret string, trunk string, additionalSubnets []string, additionalSecurityGroups []string) (*openstackprovider.OpenstackProviderSpec, error) {
```

Comment:
why the extra args when most of this is from platform and machinepool?

Reply:
totally agree! done